stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -155,20 +198,46 @@ public void createToken(final Card card, final String publishableKey, final Exec
     }
 
     private Card androidCardFromStripeCard(com.stripe.model.Card stripeCard) {
-        return new Card(null, stripeCard.getExpMonth(), stripeCard.getExpYear(), null, stripeCard.getName(), stripeCard.getAddressLine1(), stripeCard.getAddressLine2(), stripeCard.getAddressCity(), stripeCard.getAddressState(), stripeCard.getAddressZip(), stripeCard.getAddressCountry(), stripeCard.getLast4(), stripeCard.getType(), stripeCard.getFingerprint(), stripeCard.getCountry());
+        return new Card(
+                null,
+                stripeCard.getExpMonth(),
+                stripeCard.getExpYear(),
+                null,
+                stripeCard.getName(),
+                stripeCard.getAddressLine1(),
+                stripeCard.getAddressLine2(),
+                stripeCard.getAddressCity(),
+                stripeCard.getAddressState(),
+                stripeCard.getAddressZip(),
+                stripeCard.getAddressCountry(),
+                stripeCard.getLast4(),
+                stripeCard.getBrand(),
+                stripeCard.getFingerprint(),
+                stripeCard.getCountry());
     }
 
-    private Token androidTokenFromStripeToken(Card androidCard, com.stripe.model.Token stripeToken) {
-        return new Token(stripeToken.getId(), stripeToken.getLivemode(), new Date(stripeToken.getCreated() * 1000), stripeToken.getUsed(), androidCard);
+    private Token androidTokenFromStripeToken(
+            Card androidCard,
+            com.stripe.model.Token stripeToken) {
+        return new Token(
+                stripeToken.getId(),
+                stripeToken.getLivemode(),
+                new Date(stripeToken.getCreated() * 1000),
+                stripeToken.getUsed(),
+                androidCard);
     }
 
     private void tokenTaskPostExecution(ResponseWrapper result, TokenCallback callback) {
-        if (result.token != null)
+        if (result.token != null) {
             callback.onSuccess(result.token);
-        else if (result.error != null)
+        }
+        else if (result.error != null) {
```

Comment:
I don't care all that much, but isn't it closer to normal Java convention to use `} else {` instead of the version without the extra newline? It seems like we're using that style quite a bit elsewhere in the project.